docs/plugins/convert.rst

```diff
@@ -4,7 +4,9 @@ Convert Plugin
 The ``convert`` plugin lets you convert parts of your collection to a
 directory of your choice, transcoding audio and embedding album art along the
 way. It can transcode to and from any format using a configurable command
-line.
+line. This plugin is useful to convert existing tracks in your library
+to a path outside of your library. The tracks will not be removed from your
+library.
```

Comment:
Well, that's not *entirely* correct: with the `-k` option, the convert plugin will keep the new, transcoded files instead of your original files. And when it's used in "auto" mode, the plugin converts newly-imported files on the fly.

Maybe a note like this would be better located below, under "Usage," which is where we explain what the command actually does?